lua/entities/gmod_wire_expression2/core/sound.lua

```diff
@@ -70,11 +70,11 @@ local function soundStop(self, index, fade)
 	timer.Remove( "E2_sound_stop_" .. self.entity:EntIndex() .. "_" .. index )
 end
 
+
 local function soundCreate(self, entity, index, time, path, fade)
-	path = string.Trim(string.sub(path, 1, 260))
 	if path:match('["?]') then return end
```

Comment:
Can remove this if the SoundExists function is removing illegal characters

Reply:
Actually nevermind, SoundExists should return false if there are illegal characters.